css/full-width-pics.css

```diff
@@ -139,6 +139,103 @@
   font-family: sans-serif;
 }
 
+
+
+/****** Home Page Split Screen ********/
+
+.split{
+  display: flex;
+  height: 53vh;
+}
+
+.split-left{
+  width: 50%;
+  display: inline;
+  padding-top: 50px;
+  padding-right: 50px;
+  padding-left: 50px;
+  padding-bottom: 50px;
+  border-style: none solid none none;
+  border-color: #343a40;
+  border-width: 3px;
+}
+
+.split-right{
+  width: 50%;
+  padding-top: 50px;
+  padding-right: 50px;
+  padding-left: 50px;
+  padding-bottom: 50px;
+}
```

Comment:
[Response Web Design](https://blog.froont.com/9-basic-principles-of-responsive-web-design/) is a design concept for dealing with the variety of possible window sizes, layouts, and systems that could visit a web page. When examining the news feed and scaling the window down to a width that is more typical of small screens (e.g. mobile browsers), I get the following:

![image](https://user-images.githubusercontent.com/6432572/89360847-7d9ebd80-d67e-11ea-99e1-c3e7835e1ec7.png)

To correct this issue, we need to update the `.split-right` to take advantage of the [CSS Overflow Property](https://css-tricks.com/the-css-overflow-property/). By adding the `overflow: hidden` property to the `.split-right`, any overflowing news feed items will appear to slide behind the footer on the page.